posts/getting-started-with-aleo-instructions.md

```diff
@@ -138,50 +139,48 @@ function sum:
     output r2 as u32.private;
 ```
 
-First, we need to declare the program as the following:
+First, we need to declare the program as follows:
+
 ```
 program foo.aleo;
 ```
-Afterwards, we can start writing its functions (or other aleo structures such as interfaces, records, closures, as we will see later)
 
-In the case of functions we have it very easy:
+Then, we can start writing its functions (or other aleo structures such as interfaces, records, closures, as we will see later). In the case of functions we have it very easy:
 
 ```
 function [function_name]:
 ```
 
 The functions are composed of three main parts:
 
-* **The input section**
-Here we declare its input parameters:
+* **The input section**. Here we declare its input parameters:
 ```
     input r0 as u32.public;
     input r1 as u32.private;
 ```
-Everything in aleo instructions are declared/stored inside a register with a type (`i8`,`field`,`bool`, etc) and a visibility option (`public` or `private`), register are named as `r0`, `r1`, ..., `rn`. We will dig further into registers, types and visibility later on.
+Everything in Aleo instructions is declared/stored inside a register with a type (`i8`,`field`,`bool`, etc) and a visibility option (`public` or `private`). Registers are named as `r0`, `r1`, ..., `rn`. We will dig further into registers, types and visibility later on.
+
+So, in this case, we use `r0`, and `r1` to store the inputs passed in sequential order to a program as `u32` values, where we can store 32-bit unsigned integers to perform our sum operation.
 
-So in this case we use `r0`, and `r1` to store the inputs pased in sequential order to a program as `u32` values, where we can store 32bits unsigned integers to perform our sum operation.
 
+* **The instructions section**. The next section, consists in the core of our function. Here we call the amount of Aleo Instructions we need to make our program do what we want. For example, performing an add operation:
```

Comment:
```suggestion
* **The instructions section**. The next section, consists in the core of our function. Here we call the amount of Aleo instructions we need to make our program do what we want. For example, performing an add operation:
```

Before we used lowercase for "instructions". I suggest we unify using the most recurrent one.